openquakeplatform/openquakeplatform/static/js/irv_viewer_d3_tree.js

```diff
@@ -64,6 +64,51 @@
         var diagonal = d3.svg.diagonal()
             .projection(function(d) { return [d.y, d.x]; });
 
+        function isComputable(node) {
+            if (node.name === 'IRI') {
+                if (typeof node.children === 'undefined') {
+                    return false;
+                }
+                // check if both RI and SVI are computable
+                var areRiAndSviComputable = true;
+                for (var i = 0; i < node.children.length; i++) {
+                    if (!isComputable(node.children[i])) {
+                        areRiAndSviComputable = false;
+                    }
+                }
+                if (areRiAndSviComputable) {
+                    return true;
+                } else {
+                    return false;
+                }
+            }
+            if (node.name === 'SVI') {
+                if (typeof node.children === 'undefined') {
+                    return false;
+                }
+                // Check if all themes are computable
+                var areAllThemesComputable = true;
+                for (var i = 0; i < node.children.length; i++) {
+                    if (!isComputable(node.children[i])) {
+                        areAllThemesComputable = false;
+                    }
+                }
+                if (areAllThemesComputable) {
+                    return true;
+                } else {
+                    return false;
+                }
+            }
+            if (node.name === 'RI' || node.name === 'SVI' ) {
```

Comment:
On QGIS side we also have `|| node.type === node_types_dict.SV_THEME` in the conditions,
therefore we also check that all SVI themes have at least one indicator.
If you are not using the `node_types_dict`, I guess you should do a check like `|| node.type == 'SVI_THEME'` or something similar